File: src/test/java/integration/auth/PasswordEncoderTest.java
package integration.auth;

import org.junit.Test;

import com.matag.admin.MatagAdminWebSecurityConfiguration;

public class PasswordEncoderTest {
  @Test
  public void encodePassword() {
    var matagAdminWebSecurityConfiguration = new MatagAdminWebSecurityConfiguration();
    var passwordEncoder = matagAdminWebSecurityConfiguration.passwordEncoder();
    var encodedPassword = passwordEncoder.encode("password");
    System.out.println(encodedPassword);
  }
}

File: src/test/java/integration/auth/PasswordEncoderTest.kt
package integration.auth

import com.matag.admin.MatagAdminWebSecurityConfiguration
import org.junit.jupiter.api.Test

class PasswordEncoderTest {
    @Test
    fun encodePassword() {
        val matagAdminWebSecurityConfiguration = MatagAdminWebSecurityConfiguration()
        val passwordEncoder = matagAdminWebSecurityConfiguration.passwordEncoder()
        val encodedPassword = passwordEncoder.encode("password")
        println(encodedPassword)
    }
}
